IBX-1440: Add description metadata (#133)
ezsystems/user-documentation: docs/publishing/publishing.md

---

> description: You can instantly publish and newly created Content item or save its draft for later editing. Drafts are autosaved to prevent you from losing your work.

Reviewer: ```suggestion
description: You can instantly publish a newly created Content item or save its draft for editing. Drafts are autosaved to prevent you from losing your work.
```
